josh-core/src/query.rs

```diff
@@ -41,8 +54,26 @@ impl GraphQLHelper {
             variables.insert(k.to_string(), juniper::InputValue::scalar(v.render()));
         }
 
```

Comment:
maybe a comment here to explain that one case is josh-filter and the other is josh-proxy?